int32[] xmin
int32[] xmax
int32[] ymin
int32[] ymax
int8[] clsID
float32[] confidence
Distance[] distance